docs/changes.rst

```diff
@@ -17,6 +17,10 @@ Released: not yet
 * The PYWBEMCLI_TERMWIDTH environment variable was renamed to
```

Comment:
The PR contains two commits, the description of one of which does not match the PR at all. however, the overall PR content is ok, so I think what is needed is a rebase to master, squashing the two commits into one, and cleaning up the resulting commit message.

Reply:
WHOOPS.  I am not going to try to explain that one.  Removed.

Reply:
Whoops.